doc/cabal-package.rst

```diff
@@ -1454,24 +1435,25 @@ system-dependent values for these fields.
 
 .. pkg-field:: build-depends: library list
 
-    Declares the *library* dependencies required to build the current
-    package component; see :pkg-field:`build-tool-depends` for
-    declaring build-time *tool* dependencies. External library
-    dependencies should be annotated with a version constraint.
+    Declares the *library* dependencies (from the same package or other
+    packages) required to build the current package component. See
```

Comment:
```suggestion
    Declares the *library* dependencies required to build the current package component. See
```
I don't think this parenthesis is adding any extra information. You are saying either X or not X, so that's everything

Reply:
I think it is useful to be express that a component of a package can depend on other (library) components in the same package AND library components in other packages (i.e. on 'everything').

(The fact that an executable can depend on the library in the same package is the 'Cabal 1.8.0.4' feature that was highlighted in the opening sentences of this part of the guide.)

Reply:
I have accepted this, as the point I wanted to make express is now expressed elsewhere.